src/Handlers/GeneralUiStateHandler.php

```diff
@@ -93,6 +93,7 @@ protected function _getUiConfig($formatsConfig, $linksConfig)
     {
         return [
             'timezone' => $this->_getWebsiteTimezone(),
+            'currency' => $this->_getWebsiteCurrency(),
```

Comment:
Later on, move this somewhere else. It should look like every service has its own currency; see [card](https://github.com/orgs/RebelCode/projects/2#card-11147247).